lib/service_provider.rb

```diff
@@ -9,14 +9,18 @@ def initialize(host)
   end
 
   def metadata
-    ignored_methods = [:metadata, :encrypt_responses?, :encryption_opts]
+    ignored_methods = [:metadata, :encrypt_responses?, :encryption_opts, :issuer]
     metadata_methods = self.class.instance_methods(false) - ignored_methods
 
     @metadata ||= metadata_methods.inject({}) do |hash, method|
       hash.merge(:"#{method}" => send(method))
     end
   end
 
+  def issuer
+    @host
```

Comment:
Why is this needed?  Can't @host already be accessed?  Should @host be renamed to issuer for consistency?